affiliated/index.html

```diff
@@ -119,17 +119,18 @@ <h1 id="infrastructure-packages">Astropy Infrastructure Packages<a class="parali
 
 	<h1 id="installing-affiliated-packages">Installing Affiliated and Coordinated Packages<a class="paralink" href="#installing-affiliated-packages" title="Permalink to this headline">¶</a></h1>
 
-	<p>  The simplest way to install and keep up-to-date most affiliated packages is to use the <a href="https://www.continuum.io/why-anaconda">Anaconda python distribution</a>.  This distribution includes the <a href="http://astropy.readthedocs.io/en/stable/index.html">Astropy core package</a> already built in, and you can then easily install or update affiliated packages using the <a href="https://anaconda.org/astropy/packages">Astropy Conda channel</a> of binary installable packages.  Once you have Anaconda installed, you can do <code>conda search --channel astropy some_affiliated_package</code>.  Most importantly, you can then install them with <code>conda install --channel astropy some_affiliated_package</code>.  The Astropy channel is maintained by the Astropy Project on Github via code in the <a href="https://github.com/astropy/conda-channel-astropy">conda-channel-astropy</a> repository. </p>
-
-	<p> If you do not have Anaconda or wish to install from source, for most affiliated packages, downloading the source code and doing <code>python setup.py install</code> will work.  Many also support the Astropy core package's additional build and install options, as they use the affiliated package template (detailed more in the <a href="#affiliated-instructions">Becoming an Affiliated Package</a> section). That said, affiliated packages are developed independently of the Astropy core library.  This means they are free to develop their packages as they see fit, and can have a variety of different requirements or  unusual install procedures.  Hence you should refer to the package's documentation first if you encounter problems.</p>
+	<p>All packages should be available on PyPI. Some are available via conda, particularly
+	through the conda-forge channel. Always consult the individual package installation
+  instructions first before installing it. The Astropy Project is not responsible
+  for broken installation for the packages that it does not directly maintain.</p>
```

Comment:
This version feels a bit like we are trying to weasel out of something. How about:
```suggestion
	through the conda-forge channel.  That said, affiliated packages are developed 
	independently of the Astropy core library.   You should refer to the package's
	 documentation first if you encounter problems.</p>
```
After all, just because we are not responsbile for them, that does not mean we won't ever answer relevant questions about them.